### PR TITLE
Update Python documentation

### PR DESCRIPTION
**Pull request by marcellp**
# Background
#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -1,26 +1,41 @@
...
+You should be prompted to run the code in a container. If not, hit `Cmd+Shift+P` and search for
+`Remote Containers: Reopen in Container`. Selecting this item will cause the Python worker to
+reopen in a container for development.
 
-## Run locally
+### Tests
+While having the workspace running in a container, open a terminal in VS code. Go to `src/` and run:
 
-### 1. Set up environment
+    gunzip -k ../../data/test/python.h5ad
```
Comment (ivababukova) [inline python/README.md]: I never did that step .. is it necessary?

File: python/README.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-* `GITLAB_DEPLOY_USER` and `GITLAB_DEPLOY_PASSWORD`. These need to specify a valid GitLab deploy key that has read rights to the Docker registry. This secret is used by dynamically generated jobs to automatically be able to pull the worker image.
-* `K8S_SECRET_AWS_ACCESS_KEY_ID`, `K8S_SERET_AWS_SECRET_ACCESS_KEY`, `K8S_SECRET_AWS_DEFAULT_REGION`, which specify the IAM account and default region to be used with this worker.
+See the main README for instructions on how to run the workers in Docker.
 
-# Development
+### Process tasks
+Tasks are automatically processed when they are received from the SQS queue specified.
 
-## Run locally
+For **local development**, make sure you have InfraMock running alongside the `ui` and `api`. Tasks should
```
Comment (ivababukova) [inline python/README.md]: put links to inframock, the api and the ui and tell them to see their readmes on how to set them up

File: python/README.md [hunk 3]
```diff
@@ -32,20 +32,41 @@ In order to push work to the worker you can either:
...
+    code python.code-workspace
+
+You should be prompted to run the code in a container. If not, hit `Cmd+Shift+P` and search for
+`Remote Containers: Reopen in Container`. Selecting this item will cause the Python worker to
+reopen in a container for development.
+
+### Tests
+While having the workspace running in a container, open a terminal in VS code. Go to `src/` and run:
+
+    gunzip -k ../../data/test/python.h5ad
```
Comment (ivababukova) [inline python/README.md:63]: is step `gunzip -k ../../data/test/python.h5ad` necessary? I don't remember having to do it

Comment (marcellp) [reply python/README.md:63]: You need to do it the first time, after that it won't do anything. By default the files are gzipped on GitHub, so you need to unpack them.

File: python/README.md (hunk 2 above)
Comment (marcellp) [reply python/README.md]: Done.

File: python/README.md (hunk 3 above)
Comment (ivababukova) [reply python/README.md:63]: I see, cool!

**Review by ivababukova — APPROVED** [review]

**Pull request merged**